CMakeLists.txt

```diff
@@ -25,7 +25,9 @@ if( GCC )
 endif()
 if(BUILD_WITH_CLANG)
   set(CMAKE_CXX_COMPILER "${BUILD_WITH_CLANG}/bin/clang++")
```

Comment:
Why not change this line too?

Reply:
Sorry. This line needs to be removed. I'll do it if the change is fine.

Reply:
Yes, the CMakeList change is fine if you remove the duplicate assignment.

Reply:
Thank you. I'll do an amend to the PR.